src/DssSpell.sol

```diff
@@ -18,49 +18,258 @@ pragma solidity 0.6.12;
 
 import "dss-exec-lib/DssExec.sol";
 import "dss-exec-lib/DssAction.sol";
+import "lib/dss-interfaces/src/dss/GemJoinAbstract.sol";
+import "lib/dss-interfaces/src/dss/IlkRegistryAbstract.sol";
+import "lib/dss-interfaces/src/dapp/DSTokenAbstract.sol";
+import "./CentrifugeCollateralValues.sol";
 
-interface Bumpable {
-    function bump(bytes32, uint256) external;
+interface Initializable {
+    function init(bytes32) external;
+}
+
+interface Hopeable {
+    function hope(address) external;
+}
+
+interface Kissable {
+    function kiss(address) external;
+}
+
+interface RwaLiquidationLike {
+    function ilks(bytes32) external returns (string memory,address,uint48,uint48);
+    function init(bytes32, uint256, string calldata, uint48) external;
 }
 
 contract DssSpellAction is DssAction {
 
     // Provides a descriptive tag for bot consumption
     // This should be modified weekly to provide a summary of the actions
-    // Hash: seth keccak -- "$(wget https://raw.githubusercontent.com/makerdao/community/6d2eaab1360395916b9b09c4137c5da6f2de9f1a/governance/votes/Executive%20vote%20-%20July%2017%2C%202021.md -q -O - 2> /dev/null)"
-    string public constant override description =
-        "2021-07-17 MakerDAO Executive Spell | Hash: 0x887d27e20f47d0701d1eea14045bb5fdbea634401f7dc5aedd729af13a5c8ddc";
+    // Hash: seth keccak -- "$(wget https://raw.githubusercontent.com/makerdao/community/2a7a8c915695b7298fe725ee3dc6c613fa9d9bbe/governance/votes/Executive%20vote%20-%20April%2012%2C%202021.md -q -O - 2>/dev/null)"
+    string public constant override description = "RWA003-RWA006 integration";
```

Comment:
Content hash: `0xca63230792c2ab6f63397d4e48c723afedbb8b3fafb9cc1ef441e09b7d143d3c`

Reply:
```suggestion
    string public constant override description = "2021-07-17 MakerDAO Executive Spell | Hash: 0xca63230792c2ab6f63397d4e48c723afedbb8b3fafb9cc1ef441e09b7d143d3c";
```

Reply:
I updated the hash to what @brianmcmichael suggested and changed the date to `2021-07-26` because that is when governance is going to put this in the voting portal.  Governance is giving the new silver and flap change a chance for VCs to vote on.  Not sure if this is general apathy, or if they were all in Paris.